src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -652,6 +652,11 @@ public function get(string $key, mixed $default = null): mixed
         return $default;
     }
 
+    public function isStateless(): bool
```

Comment:
Just, what about commenting where such internally data is defined to be very precise ?
Appart from that, it makes sens to me as commented previously 
👌🏻

Reply:
> what about commenting where such internally data is defined to be very precise ?

Sorry I don't understand, what do you mean ?

Reply:
I meant the attribute _stateless, commenting where does it comes from internally

Reply:
I'm not even sure where it comes from.
Can be the firewall config, the routing config, can be set manually, can be coming from another request...

There is multiple `$request->attributes->set('_stateless', true)` calls in the code base

Reply:
Copy that, lets wait other reviews :)

Reply:
Do we want to also create a `setStateless()` method to have only one place where we deal with the magic `_stateless` string?

Reply:
I thought about it, but currently there is still some magic `_stateless` I dunno how to solve

For instance the FirewallMap has the code
```
if ($context->getConfig()?->isStateless() && !$request->attributes->has('_stateless')) {
     $request->attributes->set('_stateless', true);
}
```

How would I replace the `has('_stateless')` check ?

In the same way we have
```
if ($request->attributes->has('_stateless')) {
     $subRequest->attributes->set('_stateless', $request->attributes->get('_stateless'));
}
```
in the InlineFragmentRenderer

Reply:
this code should still check for the attribute directly, to distinguish from an explicitly configured `false` in route defaults vs the default value